documentation/source/user_section/tutorials.rst

```diff
@@ -76,5 +77,6 @@ If you would prefer to learn how to use SCT in a guided workshop setting, we pro
    tutorials/atlas-based-analysis
    tutorials/diffusion-weighted-mri
    tutorials/processing-fmri-data
+   tutorials/spinal-nerve-rootlets-segmentation
```

Comment:
What if we instead did something like this...

Before:

![image](https://github.com/user-attachments/assets/39f7f59a-7dc5-4a21-b4f5-6375de654244)

After:

![image](https://github.com/user-attachments/assets/52ac627a-5b35-4353-8170-87c96acc771e)

And instead of a whole new tutorial, we simply put 1 extra page at the end of the existing vertebral labeling tutorial as a sort of "bonus". But, the top-level link (shown above) would go directly to that last page.

This way, we don't have to introduce an entirely new tutorial (since the vertebral labeling tutorial uses the same `t2.nii.gz` image), but we would still provide visibility to your model in the relevant section?

Reply:
These are good points! I agree that the rootlets tutorial is reatively short and might thus be integrated into some existing tutorials. 
However, I'm not sure whether the [existing vertebral labeling tutorial](http://localhost:63342/spinalcordtoolbox/documentation/build/html/user_section/tutorials/vertebral-labeling.html) is the best choice. You're right that rootlet segmentation is related to labelling. But ideally, in future, we would like to have the possibility to label the spinal cord based on rootlets to get so-called spinal levels. So it would be actually a second possible labelling option (i.e., Opt. 1: vertebral labeling (based on bony vertebrae), Opt. 2: spinal level labeling (based on rootlets)).
Maybe a new category for all `sct_deepseg's` models would be more appropriate, as actually proposed in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4630.

Reply:
> You're right that rootlet segmentation is related to labelling. But ideally, in future, we would like to have the possibility to label the spinal cord based on rootlets to get so-called spinal levels. So it would be actually a second possible labelling option (i.e., Opt. 1: vertebral labeling (based on bony vertebrae), Opt. 2: spinal level labeling (based on rootlets)). From this perspective, I would not mix rootlets with vert labeling for now.

Ahh! Thank you for clarifying the future plans of this model. That makes sense to me. :)

> Maybe a new category for all `sct_deepseg's` models would be more appropriate, as actually proposed in #4630.

I think this makes sense to me!!

Although, the linked PR only includes a model gallery on the `sct_deepseg` page itself. I would still want to make sure there is visibility for the models on the tutorials page as well? In that case, maybe we should consider something like this:

![image](https://github.com/user-attachments/assets/fc256c86-ee82-4a82-9b3d-523fede0afe2)

And simply use the sub-links to jump to the existing documentation page found [here](https://spinalcordtoolbox.com/user_section/tutorials/segmentation/sct_deepseg.html), then update it so that it includes a usage example or two?

Reply:
> Although, the linked PR only includes a model gallery on the `sct_deepseg` page itself. I would still want to make sure there is visibility for the models on the tutorials page as well? In that case, maybe we should consider something like this:
> 
> ![image](https://private-user-images.githubusercontent.com/16181459/369936697-fc256c86-ee82-4a82-9b3d-523fede0afe2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjc0NTg0NTgsIm5iZiI6MTcyNzQ1ODE1OCwicGF0aCI6Ii8xNjE4MTQ1OS8zNjk5MzY2OTctZmMyNTZjODYtZWU4Mi00YTgyLTliM2QtNTIzZmVkZTBhZmUyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTI3VDE3MjkxOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWVmYjcxMjUzZjdiNDMyMDNkNGUwNTQ1N2FmMzMwMzczNzA3ZWJjOWI5MDI3Zjg0OWJhZGU0NGIwNTIyYjYxZTEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.D93qI-SrSklnjLmkq8mYeXBkqMJZVzX0QdxG47F4pqE)

This is a good idea! I am thinking, though—since we have quite a few `sct_deepseg` models, grouping everything under one broad category like "Specialized models (nerve rootlets, lesions)" might make it harder to navigate. How about breaking it down a bit more, like this:

```
1. Segmentation
    - Spinal cord segmentation 
    - Lesions segmentation
    - Nerve rootlets segmentation
    - Tumors segmentation
    - ...
```

Reply:
Side note when looking at [this page](https://spinalcordtoolbox.com/user_section/tutorials/segmentation/sct_deepseg.html): we can add nnUNet and MONAI to the [first sentence](https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/07a65d26f6178f8f1e07f149d96a1ce0c1051109/documentation/source/user_section/tutorials/segmentation/sct_deepseg.rst?plain=1#L4).

Reply:
> everything under one broad category like "Specialized models (nerve rootlets, lesions)" might make it harder to navigate. How about breaking it down a bit more, like this:

I like this suggestion!

Although, I think even if we have multiple "Table of Contents" entries, maybe we should still just point all of them to the same page? (I suggest this because the only real difference between lesion seg/rootlets seg/tumors seg/etc. is the task name... otherwise, they all have the same syntax? Meaning we don't really need duplicate "Before starting this tutorial" pages and etc.)

So, for example, in the context of the SCT course, all of the different models can pretty much be summed up by a single slide (for the syntax) and maybe a second slide showcasing the model gallery?

Reply:
> I suggest this because the only real difference between lesion seg/rootlets seg/tumors seg/etc. is the task name... otherwise, they all have the same syntax? 

Yes, you are right. The syntax is the same; only the task name changes.

> Meaning we don't really need duplicate "Before starting this tutorial" pages and etc.)

Hmm, good point! The thing is that some models require different sample data. For example, the SCIseg model requires the [T2w image with fake lesion](https://spinalcordtoolbox.com/user_section/tutorials/lesion-analysis/before-starting.html), meaning that the `cd` command is slightly different. Compare the following.

Rootlets:
```
cd {PATH_TO_DOWNLOADED_DATA}/single_subject/data/t2/
```

SCIseg:
```
cd {PATH_TO_DOWNLOADED_DATA}/single_subject/data/t2_lesion/
```

Reply:
> So, for example, in the context of the SCT course, all of the different models can pretty much be summed up by a single slide (for the syntax) and maybe a second slide showcasing the model gallery?

I like this idea!

Reply:
> for example, the SCIseg model requires the [T2w image with fake lesion](https://spinalcordtoolbox.com/user_section/tutorials/lesion-analysis/before-starting.html)

Ahh, very good point. That does complicate things... I had proposed the above idea thinking we could use the existing `t2.nii.gz` file for everything. :P 

This is very tricky! To summarize:

- We want each region type (lesion, tumor, gm, rootlets) to have its own visible section in the Table of Contents, so that users can easily discover the new models from the "Tutorials" page.
- Yet, it kind of doesn't make sense to have unique tutorials for each of the different `sct_deepseg` tasks, since the underlying syntax is very simple, and is the same for all `sct_deepseg` tasks. As long as they have the basic syntax `sct_deepseg {TASK_NAME} -i input.nii.gz`, they don't really need any further instruction.
- But, because different tasks need different data, each tutorial would need its own .zip download, meaning that it would be awkward to create a combined "sct_deepseg" page/tutorial.

OK. I have one more idea. What if we don't create individual `sct_deepseg` tutorials at all, and in PR https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4630 we instead create links from `tutorials.rst` to `command-line/sct_deepseg.rst`? 

Sort of like this:

1. [Spinal cord segmentation](https://spinalcordtoolbox.com/user_section/tutorials/segmentation.html)
   - [Specialized cord segmentation (lumbar, EPI, mouse)](https://spinalcordtoolbox--4630.org.readthedocs.build/user_section/command-line/sct_deepseg.html#spinal-cord)
   - [Specialized GM/WM segmentation (7T T2*, ex vivo, mouse)](https://spinalcordtoolbox--4630.org.readthedocs.build/user_section/command-line/sct_deepseg.html#gray-matter)
   - [Pathology segmentation (tumors, MS lesions, SCI lesions)](https://spinalcordtoolbox--4630.org.readthedocs.build/user_section/command-line/sct_deepseg.html#pathologies)
   - [Other structures (rootlets)](https://spinalcordtoolbox--4630.org.readthedocs.build/user_section/command-line/sct_deepseg.html#other-structures)
2. [Vertebral labeling](https://spinalcordtoolbox.com/user_section/tutorials/vertebral-labeling.html)
3. [Shape analysis](https://spinalcordtoolbox.com/user_section/tutorials/shape-analysis.html)

And all of those sub-section links directly go to `sct_deepseg`. (Perhaps we can add a basic usage command to each section, too.)

Reply:
> Sort of like this:
> 
> > 1. [Spinal cord segmentation](https://spinalcordtoolbox.com/user_section/tutorials/segmentation.html)
> >    
> >    * [Specialized cord segmentation (lumbar, EPI, mouse)](https://spinalcordtoolbox--4630.org.readthedocs.build/user_section/command-line/sct_deepseg.html#spinal-cord)
> >    * [Specialized GM/WM segmentation (7T T2*, ex vivo, mouse)](https://spinalcordtoolbox--4630.org.readthedocs.build/user_section/command-line/sct_deepseg.html#gray-matter)
> >    * [Pathology segmentation (tumors, MS lesions, SCI lesions)](https://spinalcordtoolbox--4630.org.readthedocs.build/user_section/command-line/sct_deepseg.html#pathologies)
> >    * [Other structures (rootlets)](https://spinalcordtoolbox--4630.org.readthedocs.build/user_section/command-line/sct_deepseg.html#other-structures)
> > 2. [Vertebral labeling](https://spinalcordtoolbox.com/user_section/tutorials/vertebral-labeling.html)
> > 3. [Shape analysis](https://spinalcordtoolbox.com/user_section/tutorials/shape-analysis.html)

This sounds good! 
Maybe instead of `1. Spinal cord segmentation`, we could use more general `1. Segmentation` because the section contains not only spinal cord segmentation (e.g., tumors, lesions, rootlets):

> 1. [Segmentation](https://spinalcordtoolbox.com/user_section/tutorials/segmentation.html)
>    
>    * [Spinal cord segmentation (contrast-agnostic, lumbar, EPI, mouse)](https://spinalcordtoolbox--4630.org.readthedocs.build/user_section/command-line/sct_deepseg.html#spinal-cord)
>    * [GM/WM segmentation (7T T2*, ex vivo, mouse)](https://spinalcordtoolbox--4630.org.readthedocs.build/user_section/command-line/sct_deepseg.html#gray-matter)
>    * [Pathology segmentation (tumors, MS lesions, SCI lesions)](https://spinalcordtoolbox--4630.org.readthedocs.build/user_section/command-line/sct_deepseg.html#pathologies)
>    * [Other structures (rootlets)](https://spinalcordtoolbox--4630.org.readthedocs.build/user_section/command-line/sct_deepseg.html#other-structures)
> 2. [Vertebral labeling](https://spinalcordtoolbox.com/user_section/tutorials/vertebral-labeling.html)
> 3. [Shape analysis](https://spinalcordtoolbox.com/user_section/tutorials/shape-analysis.html)